packages/cli/src/oclif/commands/invoke.js

```diff
@@ -1061,7 +1061,7 @@ class InvokeCommand extends BaseCommand {
   async perform() {
     let authId = this.flags['authentication-id'];
 
-    const dotenvResult = dotenv.config({ override: true });
+    const dotenvResult = dotenv.config({ override: true, quiet: true });
```

Comment:
The breaking change in dotenv 17+ was the addition of logs that broke JSON parsers ( https://github.com/motdotla/dotenv/blob/master/CHANGELOG.md ). The recommended fix is to use `quiet: true` for similar behavior as before